src/forms/textarea.story.jsx

```diff
@@ -51,7 +51,7 @@ storiesOf('Textarea', module)
 			<InfoWrapper>
 				<Bounds><Section>
 					<Textarea
-						rows='auto'
+						autosize
```

Comment:
I discussed with Mike, we don't want to default to `rows={1}` because there is a case where you don't want to set `rows` at all.